ansible_collections/arista/avd/extensions/molecule/eos_cli_config_gen/inventory/host_vars/host1/snmp-server.yml

```
@@ -11,6 +11,12 @@ snmp_server:
       - id: DEADBEEFCAFE123456
         address: 2.2.2.2
         udp_port: 1337
+      # These next two are needed to configure non localized remote users defined below
+      - id: 424242424242DEAD
+        address: 42.42.42.42
+      - id: 424242424242DEAD666
```

```suggestion
      - id: 424242424242DEAD6666
```

We should use even number of HEX characters to avoid any side effects on the EOS (like adding `0` in `[-2]` position in the later example) and simplify further testing of the produced config against EOS

```
veos-4-33-0f#conf session 
veos-4-33-0f(config-s-s34c)#snmp-server engineID remote 42.42.42.42 424242424242DEAD
veos-4-33-0f(config-s-s34c)#show session-config diffs 
--- system:/running-config
+++ session:/s34c-session-config
+snmp-server engineID remote 42.42.42.42 424242424242DEAD
veos-4-33-0f(config-s-s34c)#snmp-server engineID remote 42.42.42.42 udp-port 666 424242424242DEAD666
veos-4-33-0f(config-s-s34c)#show session-config diffs
--- system:/running-config
+++ session:/s34c-session-config
+snmp-server engineID remote 42.42.42.42 424242424242DEAD
+snmp-server engineID remote 42.42.42.42 udp-port 666 424242424242DEAD6606
veos-4-33-0f(config-s-s34c)#
```